package.json

```diff
@@ -198,8 +198,10 @@
     "prepare": "husky",
     "preinstall": "./scripts/preinstall.sh && npx solidarity",
     "start": "react-native start",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
+    "test": "jest --runInBand",
+    "test:ci": "jest --verbose false --forceExit",
+    "test:ci:coverage": "jest --coverage --verbose false --forceExit",
```

Comment:
On Github Action, --forceExit should be used here because I think it's better to force jest to exit even if there's leaky tests, or unclose database connection.  This should be done and fixed locally.

This is just my opinion.